src/assets/css/global.css

```diff
@@ -151,12 +151,14 @@ textarea:focus {
   display: flex;
   align-items: center;
   justify-content: center;
+  flex-wrap: wrap;
 }
 
 .flex-items a {
   display: block;
-  margin: 0 20px;
+  margin: 20px;
   transition: all 0.4s ease-in-out; 
+  width: 350px;
```

Comment:
should we work here with relativ metrics?

Reply:
Yes, that would be better. Can you suggest the changes ?